addons/delivery/models/stock_picking.py

```diff
@@ -83,6 +97,21 @@ def _prepare_shipping_invoice_line(self, picking, invoice):
 
         return res
 
+    @api.model
+    def _invoice_create_line(self, moves, journal_id, inv_type='out_invoice'):
+        InvoiceLine = self.env['account.invoice.line']
+        invoice_ids = super(StockPicking, self)._invoice_create_line(moves, journal_id, inv_type=inv_type)
+        delivey_invoices = {}
+        for move in moves:
+            for invoice in move.picking_id.sale_id.invoice_ids.filtered(lambda invoice: invoice.id in invoice_ids):
+                delivey_invoices[invoice] = move.picking_id
+        if delivey_invoices:
+            for invoice, picking in delivey_invoices.items():
+                invoice_line = self._prepare_shipping_invoice_line(picking, invoice)
```

Comment:
you should call method: picking._prepare_shipping_invoice_line(invoice). Please remove picking object from argument of _prepare_shipping_invoice_line.